### PR TITLE
[bulk] Validation: altText missing

### PR DESCRIPTION
**Pull request by TimShererWithAquent**

### CODE REVIEW

File: iis/develop/using-webmatrix/make-your-website-seo-friendly.md [hunk 1]
```diff
@@ -17,15 +17,15 @@ After you've created a website, you can use WebMatrix Beta to run an analysis to
 
 To get started, open an existing website in WebMatrix. Then click **Run a report** or **Reports**.
 
-[![](make-your-website-seo-friendly/_static/image3.png)](make-your-website-seo-friendly/_static/image1.png)
+[![Screenshot shows the WebMatrix window with displaying MyNewSite, with Reports selected and the Run a report link highlighted.](make-your-website-seo-friendly/_static/image3.png)](make-your-website-seo-friendly/_static/image1.png)
```
Comment (MeeraDi) [inline iis/develop/using-webmatrix/make-your-website-seo-friendly.md:20]: Screenshot shows the WebMatrix window displaying MyNewSite, with Reports selected and the Run a report link highlighted.

File: iis/extensions/configuring-application-request-routing-arr/manually-override-cache-control-directives-using-application-request-routing.md [hunk 2]
```diff
@@ -62,7 +62,7 @@ This walkthrough assumes that ARR is already configured with disk cache. Review
     > [!NOTE]
     > Both the Host name and the URL are optional fields.
 
-    ![](manually-override-cache-control-directives-using-application-request-routing/_static/image3.jpg)
+    ![Screenshot shows teh Add Cache Control Rule dialog box with values added.](manually-override-cache-control-directives-using-application-request-routing/_static/image3.jpg)
```
Comment (MeeraDi) [inline iis/extensions/configuring-application-request-routing-arr/manually-override-cache-control-directives-using-application-request-routing.md:65]: Screenshot shows the Add Cache Control Rule dialog box with values added.

File: iis/extensions/installing-application-request-routing-arr/install-application-request-routing-version-2.md [hunk 3]
```diff
@@ -103,8 +103,8 @@ In this step, you will disable the Idle Time-Out under the application pool proc
 1. Launch IIS Manager.
 2. Select **Application Pools**.  
   
-    [![](install-application-request-routing-version-2/_static/image3.jpg)](install-application-request-routing-version-2/_static/image2.jpg)
-3. By default, **DefaultAppPool** is the corresponding application pool for the **Default Web Site**. Select **DefaultAppPool**. In the **Actions** pane, under **Edit Application Pool**, click **Advanced Settings…**[![](install-application-request-routing-version-2/_static/image5.jpg)](install-application-request-routing-version-2/_static/image4.jpg)
+    [![Screenshot shows the I I S Manager navigation view with a Application Pools selected.](install-application-request-routing-version-2/_static/image3.jpg)](install-application-request-routing-version-2/_static/image2.jpg)
```
Comment (MeeraDi) [inline iis/extensions/installing-application-request-routing-arr/install-application-request-routing-version-2.md:106]: Screenshot shows the I I S Manager navigation view with Application Pools selected.

File: iis/extensions/installing-application-request-routing-arr/install-application-request-routing-version-2.md [hunk 4]
```diff
@@ -126,8 +126,8 @@ Similarly, disable the application pool recycle setting so that the application
 1. Launch IIS Manager.
 2. Select **Application Pools**.  
   
-    [![](install-application-request-routing-version-2/_static/image7.jpg)](install-application-request-routing-version-2/_static/image6.jpg)
-3. By default, **DefaultAppPool** is the corresponding application pool for the **Default Web Site**. Select **DefaultAppPool**. In the **Actions** pane, under **Edit Application Pool**, click **Recycling…**[![](install-application-request-routing-version-2/_static/image9.jpg)](install-application-request-routing-version-2/_static/image8.jpg)
+    [![Screenshot shows the navigation view with a Application Pools selected.](install-application-request-routing-version-2/_static/image7.jpg)](install-application-request-routing-version-2/_static/image6.jpg)
```
Comment (MeeraDi) [inline iis/extensions/installing-application-request-routing-arr/install-application-request-routing-version-2.md:129]: maybe "an Application Pools"

**Comment by TimShererWithAquent** [comment]
@Rick-Anderson @mcleblanc This PR fixes validation warnings/suggestions per a platform-wide push to increase content health scores. Please review at your convenience and sign off or merge if you approve, or let me know if I need to make any other changes. cc: @MonicaRush @snoviking

**Review by Rick-Anderson — APPROVED** [review]

**Pull request merged**